internal/host/transition_test.go

```diff
@@ -2050,8 +2050,12 @@ var _ = Describe("Refresh Host", func() {
 				Expect(db.Create(&otherHost).Error).ShouldNot(HaveOccurred())
 				cluster = getTestCluster(clusterId, t.machineNetworkCidr)
 				Expect(db.Create(&cluster).Error).ToNot(HaveOccurred())
+				expectedSeverity := models.EventSeverityInfo
```

Comment:
Why not use common.GetEventSeverityFromHostStatus(t.dstState) instead of setting the severity here?

Reply:
I didn't want to use the code to check the code